contents/handbook/strategy/product-prioritization.md

```diff
@@ -0,0 +1,58 @@
+
+## How do we prioritize?
+
+Products we build into the platform should:
+
+* be recognizable/positioned as products that our ICP already uses
+* use or add to customer or product data
+* improve our other products
+* be designed to enable us to get in first with that product (even if in practise we often rip and replace an existing product)
+* help customers build more successful products (this doesn't _just_ mean writing code, it means commercial stuff too)
+* ideally work with our snippet
+
+Whilst it's possible for others to build on our platform, our priorities are:
+
+* build major new products ourselves, and improve our own ability to ship each new product (ie standard ways to launch, bill, market them and so on)
+* make our existing products headless/with good APIs, where relevant - this helps us to reuse existing work and to reduce technical debt _and_ they help the community achieve the following:
+* enable the community to build hacky internal tools or small bootstrapped products via API
+
+What we want to avoid:
+
+* We should avoid hosting other people's products for them. We tried this with apps/plugins and it was painful from a security perspective. We should instead encourage API building.
+* We shouldn't expect VC backed startups to build huge products on our platform for a very long time. This may / may not come much later on when we have the distribution to support this.
+
+We must be able to get in first for any of our tools. We don't want to have to replace a complex stack, we should be first in, and should therefore be the default for choice for any new tool.
+
+## What makes a tool in-scope or out-of-scope?
+
+The earlier stage a company is, the more likely that they don't have specialist organizations by role, such as engineering, marketing, product, data, customer success, sales.
+
+The more engineering-led the organization, the stronger a fit PostHog is for the customer, and the more likely it is that product engineers are driving all of the above areas. For example, a YC backed startup will typically have two technical cofounders, who will handle every role in the company.
+
+Therefore, any of the above are possible for us to build if the scope within them is strictly limited to what our ICP needs. Here is a non-exhaustive and rough list, sorted by stack, of tools that could fit our criteria:
+
+Product stack = product tours, feedback and surveys, analytics, recordings, roadmap, beta management (self-serve feature flags), user interviewing, alerting
+Engineering stack = feature flags, error handling, APM
+Growth stack = experimentation, session recordings, funnels, lead scoring, revenue (and integration)
+Data stack = warehouse, CDP / etl / reverse etl, transformations, monitoring and observability, data governance, ML and DS tooling, headless analytics
+BI stack = notebooks, querying, visalizations, forecasting
+Marketing stack = heatmaps, scrollmaps, attribution and performance, messaging, community and referral
```

Comment:
Did we want to include the engineering focused CS tools somewhere? Agreed with the exec sprint notes that it wouldn't make sense for now however

Reply:
genuine q: should we start with tools that are close to eng then move away from product into commercial-oriented stuff, or is a more balanced approach better for our customers?

Reply:
Agree it's definitely a case of not all of the above are equal. How close it is to product/engineering is one factor (but not the only) that we should consider.

What I think we'll probably want to do here is expand on each one of these ideas a bit further including potential market size, how much would it benefit our existing tools, strategic importance, interest/suitability from the teams etc. And then categorize the ideas roughly based on timeline e.g. we want to get to these soon, after we raise next we want to build these, we likely won't get around to these for a while

Let's chat about this more when you're back

Reply:
I think the main value of this work here is to give everyone in the company a method to prioritise the next products. Since we want to encourage ideas and initiatives coming up from the teams, we probably shouldn't prioritise 'all' the products we can come up with now and stick them on a roadmap. I like the list we have above, explicitly labelled as 'non-exhaustive and rough'.

> genuine q: should we start with tools that are close to eng then move away from product into commercial-oriented stuff, or is a more balanced approach better for our customers? why/why not - feels very important

Yes, I think this is the main point I personally would like a bit more detail on. Do we want to build the tools for engineers, or the tools to build a successful product? What would allow me to prioritise building an APM vs. a simple marketing messaging tool, built for engineers? 

I don't think we should first build out the whole engineering stack and then move into other areas. I think companies like DataDog are very strong at that and what is PostHog's success is that we are leaning more towards product / enabling engineers/founders to build a holistically good product, rather than 'just' a great engineering org. This still leaves a lot of grey area though.

While trying to answer this question myself, I think we should continue to lean heavily into our mission of "helping engineers be better at product." This is something that is not explicitly mentioned in the list above "Products we build into the platform should:" Do the products we consider actually make engineers better at product, and which of the products we consider will have the highest expected impact enabling engineers building a successful product?

Reply:
My hypothesis/opinionated marketing perspective is that integration with marketing tools is way more important than specific marketing-y features. 

If we had great integrations with:
- Google Ads/Tag Manager
- Facebook Ads/Pixel
- Mailchimp
- Customer.io

that would probably overcome 80% of the objections marketers have to using PostHog. And in larger orgs, engineers will need to convince marketing to use us over Amplitude, Mixpanel. 

The rest of those features look nice, but I don't think would move the needle for marketers anywhere near as much.

Reply:
Discussed sync with Annika but writing it up for James etc.

> I think the main value of this work here is to give everyone in the company a method to prioritise the next products. Since we want to encourage ideas and initiatives coming up from the teams, we probably shouldn't prioritise 'all' the products we can come up with now and stick them on a roadmap. I like the list we have above, explicitly labelled as 'non-exhaustive and rough'.

"Give everyone in the company a method to prioritise the next products" this is definitely one of the goals and very important to keep encouraging bottom up innovation.

Another explicit goal is to enable longer-term planning (how much should we raise? which new markets should we go after? which new teams should we form?). Particularly for new products which aren't as adjacent we'll need some extra top down effort to get things moving (such as the data warehouse). Importantly though, this should have input 
from across the company to make this high level plan. So I still think this work is valid

Agreed on this "explicitly labelled as 'non-exhaustive and rough'"

> I don't think we should first build out the whole engineering stack and then move into other areas ... I think we should continue to lean heavily into our mission of "helping engineers be better at product."

Yep agreed!